src/runtime/pkg/katautils/config.go

```diff
@@ -987,16 +1002,21 @@ func newClhHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		return vc.HypervisorConfig{}, err
 	}
 
-	if image == "" && initrd == "" {
-		return vc.HypervisorConfig{},
-			errors.New("image or initrd must be defined in the configuration file")
+	igvm, err := h.igvm()
+	if err != nil {
+		return vc.HypervisorConfig{}, err
 	}
 
 	rootfsType, err := h.rootfsType()
 	if err != nil {
 		return vc.HypervisorConfig{}, err
 	}
 
+	if image == "" && initrd == "" && igvm == "" {
```

Comment:
Do we also want to explicitly validate that if we have IGVM, kernel and initrd/image may *not* be specified (mutually exclusive)?

Reply:
This is now added in hypervisor_config_linux.go